### PR TITLE
readme: make it more neutral

### PR DESCRIPTION
**Pull request by notaz**
I'd like the readme to not look weird in notaz/picodrive when I merge all the changes, where it will be unclear who the 1st person pronoun refers to. Several people have put quite some effort into it, so I think it's best not to highlight anyone.

Also remove the old readme which has become outdated.

### CODE REVIEW

**Comment by irixxxx** [comment]
I was thinking to update it anyway, so it's good timing.

I'm planning to add a list of features (supported Sega hardware, supported image formats etc.) and maybe some basic usage instructions. Also, the build instructions should be replaced by some instructions for generic builds and a reference to the tools/release script for the platform builds.

BTW, I'm heading for 2.00 for some time now. That should add some minimal keyboard support for SC-3000 (hard to do cleanly with the old SDL1) and a small UI redesign to make managing the plethora of options easier. I plan to add some kind of "global setting" switch similar to the renderer stuff, maybe "fast", "good", "accurate", to select a set of options for each of those scenarios. If you have any more suggestions I'd really appreciate them.

**Comment by notaz** [comment]
Sounds good. I like the direction you are taking the project. Still suggest merging this and then tune things.

As for options, I now dislike them and started removing them in PCSX as some people turn them on, forget and then report bugs. Performance related ones might be helpful on old devices, so I'd keep them but put somewhere deeper in menus with easy "reset to default". Although that might not be doable with libretro, IDK.

**Comment by irixxxx** [comment]
I'm no libretro buff as well. There is the possibility to structure your menus into submenus, but IDK how far you can get with that. As for resetting to default, no idea if this can be achieved with the UI. I normally just throw away the core config file.

As for the options, I was thinking to hide away most of the advanced options from the UI. If somebody must, they can add them to the config2 file, but I plan to add a warning to the UI that some hidden option is not on the default value.
Some of the speed related options (e.g. the SSG-EG one) may move to the normal option page, however I'll need to check if the performance impact on old toasters is enough to actually justify this.

In my experience everything above 1 GHz clock is running almost everything at full speed anyway even without all those asm optimizations, which is including quite new devices with rather old cpu designs like the Miyoo (arm926), or the Funkey FWIW (neat device btw), or most newer Ingenic MIPS based devices.

**Comment by notaz** [comment]
Sounds good to me.

**Comment by irixxxx** [comment]
A bit off-topic here, but since I started that discussion with @notaz:
I've pushed a new branch menu-2 with some menu revisions. Comments welcome.

Is anybody using the ability to disable parts of the sound? I'm thinking about throwing out disabling the separate sound switches and using the global sound disable switch not just for suppressing sound output, but for disabling all sound computation.

File: README.md [hunk 1]
```diff
@@ -1,21 +1,18 @@
...
+too, like SH2 recompilers for MIPS (mips32r2), ARM64 (armv8), RISC-V (RV64IM)
+and PowerPC (G4/2.03).
 
-I added support for MIPS (mips32r2), ARM64 (armv8), RISC-V (RV64IM) and
-PowerPC (G4/2.03) support to the SH2 recompiler, as well as spent much effort to
-optimize the DRC-generated code. I also optimized SH2 memory access inside the
-emulator, and did some work on M68K/SH2 CPU synchronization to fix some problems
-and speed up the emulator.
+PicoDrive was the first emulator ever to properly emulate Virtua Racing and
+it's SVP chip.
```
Comment (Clownacy) [inline README.md:8]:
```suggestion
its SVP chip.
```

**Pull request merged**